src/GMActions.cpp

```diff
@@ -45,6 +48,71 @@ void GameMasterActions::onReceiveClientCommand(int32_t client_id, sf::Packet& pa
             }
         }
         break;
+    case CMD_INTERCEPT_ALL_COMMS_TO_GM:
+        {
+            bool value;
+            packet >> value;
+            gameGlobalInfo->intercept_all_comms_to_gm = value;
+        }
+        break;
+    case CMD_CALL_GM_SCRIPT:
+        {
+            int index;
+            packet >> index;
+            int selectedItemsLeft;
+            packet >> selectedItemsLeft;
+            PVector<SpaceObject> selection;
+            while (selectedItemsLeft--) {
+                int selectedItemId;
+                packet >> selectedItemId;
+                P<SpaceObject> object;
+                if (game_server)
```

Comment:
this should always be running on the server, so the if (game_server) is alway true